zimbra_all/opt/push-logs-to-ELK/60-output.conf

```diff
@@ -0,0 +1,35 @@
+# This line sends all lines to defined IP address at port 10514,
+# using the "json-template" format template
+#
+####### server logtash config listen port 10514/udp like: /etc/logstash/conf.d/logstash.conf
+# # This input block will listen on port 10514 for logs to come in.
+# # host should be an IP on the Logstash server.
+# # codec => "json" indicates that we expect the lines we're receiving to be in JSON format
+# # type => "rsyslog" is an optional identifier to help identify messaging streams in the pipeline.
+
+# input {
+#   udp {
+#     host => "logstash_ip"
+#     port => 10514
+#     codec => "json"
+#     type => "rsyslog"
+#   }
+# }
+
+# # This is an empty filter block.  You can later add other filters here to further process
+# # your log lines
+
+# filter { }
+
+# # This output block will send all events of type "rsyslog" to Elasticsearch at the configured
+# # host and port into daily indices of the pattern, "rsyslog-YYYY.MM.DD"
+
+# output {
+#   if [type] == "rsyslog" {
+#     elasticsearch {
+#       hosts => [ "elasticsearch_private_ip:9200" ]
+#     }
+#   }
+# }
+
+*.*                         @logstash_ip:10514;json-template
```

Comment:
This should be a specific IP (of Logstash), isn't it?
So how to pass that value to? (maybe through a parameter on "docker run"?)